text/0255-object-safety.md

```diff
@@ -33,7 +33,7 @@ SomeTrait for SomeTrait { ... }`, but that seems weird and confusing and rather
 like boilerplate. Note that the precise mechanism here is out of scope for this
 RFC).
 
-This is only sound if the trait is /object-safe/. We say a method `m` on trait
+This is only sound if the trait is object-safe. We say a method `m` on trait
```

Comment:
Not sure if the forward-slashes were useful here ... perhaps no punctuation or quotation marks would be better